src/App.js

```diff
@@ -1,12 +1,35 @@
-import './App.css';
+import "./App.css";
+import React, { useState } from "react";
+import Counter from "./components/Counter";
+import SearchForm from "./components/SearchForm";
+import GenreSelect from "./components/GenreSelect";
 
 function App() {
+  var temporaryStyles = {
```

Comment:
use `const` instead of `var`

Reply:
and better to use className as in components, just to keep the same approach everywhere